src/Tasks/XmlPeek.cs

```diff
@@ -314,7 +314,7 @@ public enum XmlModes
             /// Creates correct reader based on the input type.
             /// </summary>
             /// <returns>The XmlReader object</returns>
-            public XmlReader CreateReader(bool prohibitDtd)
+            public XmlReader CreateReader(bool prohibitDtd = true)
```

Comment:
This parameter is set by the `ProhibitDtd` property (line ~71), we should also default that value to true.

Reply:
Thanks,
So I will add the below (assuming we are on C# 6.0).
```
public bool ProhibitDtd { get; set; } = true;
```
Is that right?

Reply:
Thank you for your input. 
I will remove the optional value for _prohibitDtd_ parameter.

Reply:
> Thanks,
> So I will add the below (assuming we are on C# 6.0).
> 
> ```
> public bool ProhibitDtd { get; set; } = true;
> ```
> 
> Is that right?

Yes, that looks right. So change line 71 to that and remove this default value, and we're good!